src/Microsoft.Dism/DismApi.cs

```diff
@@ -11,7 +11,29 @@ namespace Microsoft.Dism
     /// Represents the main entry point into the Deployment Image Servicing and Management (DISM) API.
     /// </summary>
     public static partial class DismApi
-    {
+    {
+        #region Properties
+
+        private static DismGeneration dismGenerationInitialized;
+
+        /// <summary>
+        /// Represents the DISM Generational Library initialized for use with the DismApi Wrapper (via InitializeEx()). Returns the specific DismGeneration in use; otherwise, returns DismGeneration.NotFound.
+        /// </summary>
+        public static DismGeneration DismGenerationInitialized

```

Comment:
Please make this internal, an autoproperty, and set a initial value so you can get rid of the `dismGenerationInitialized` field and setting it in `Initialize`

```C#
internal static DismGeneration DismGenerationInitialized { get; set; } = DismGeneration.None;
```